app/alembic/env.py

```
@@ -15,8 +15,7 @@
 sys.path.insert(0, str(BASE_DIR))
 
 from app.core.settings import settings  # noqa: E402
-from app.infrastructure.db.models.base import Base  # noqa: E402
-from app.infrastructure.db.models.vault_orm import VaultORM  # noqa: F401,E402
+from app.infrastructure.db.models import Base  # noqa: E402
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)